ja/koreader.po

```diff
@@ -10145,26 +10113,24 @@ msgstr "保存していない変更があります。"
 
 #: frontend/ui/widget/inputdialog.lua:706
 msgid "Discard"
-msgstr ""
+msgstr "破棄"
 
 #: frontend/ui/widget/inputdialog.lua:711
 msgid "Changes discarded"
-msgstr ""
+msgstr "変更は破棄されました"
 
 #: frontend/ui/widget/inputdialog.lua:773
 msgid "Find"
 msgstr "検索"
 
 #: frontend/ui/widget/inputdialog.lua:794
-#, fuzzy
 msgid "Find first"
-msgstr "ファイルを検索"
+msgstr "最初に検索"
 
 #: frontend/ui/widget/inputdialog.lua:805
 #: frontend/ui/widget/inputdialog.lua:828
-#, fuzzy
 msgid "Found in line %1."
-msgstr "'%1' に一致するファイルが見つかりません。"
+msgstr "Line '%1' に一致する結果が見つかりました。"
```

Comment:
Line? (Just taking a random peek.)

Reply:
Line  can be translated into 'Line'  or maybe it means line number. 
I am not sure about the context at this moment.
At  finally stage I  will run through all the lines  , so  just take a compromised solution to not translate this word 'Line'.

Reply:
Ah, but you removed the fuzzy. ;-)

Reply:
yeah,  I found fuzzy items will not be shown in the simulator...   :)  . 
Anyway if the criteria will not permit the removing  `fuzzy` under this circumstance, I will put it back till I found out the context.

Reply:
It should be fine, but in principle untranslated and unfinished (fuzzy) is generally what translators will go looking for.

Reply:
Got it~